app/views/users/index.html.erb

```diff
@@ -1,5 +1,22 @@
 <div class="users">
+  <% flash.each do |type, msg| %>
+    <div class="alert">
+      <% if type == 'error' then %>
+        <div class="alert_inner alert-danger">
+          <%= msg %>
+        </div>
+      <% end %>
+      <% if type == 'success' then %>
+        <div class="alert_inner alert-success">
+          <%= msg %>
+        </div>
+      <% end %>
+    </div>
+  <% end %>
   <h1>Users</h1>
+  <div class="create_post">
+    <%= link_to "Create Post", "/post/new" %>
+  </div>
```

Comment:
- [ ] This is not supposed to be here. Kindly remove it from here.

Reply:
Kindly, where should this button be @Qoosim. I am confused at at this point.